app/controllers/users_controller.rb

```diff
@@ -1,6 +1,25 @@
 class UsersController < ApplicationController
   def show
     @user = User.find(params[:id])
-    @user.parties
+  end
+
+  def new
+  end
+
+  def create
+    new_user = User.new(user_params)
+    if new_user.save
+      flash[:notice] = 'Successfully Added New User'
+      redirect_to user_path(new_user)
+    else
+      flash[:alert] = "Error: #{error_message(new_user.errors)}"
+      render 'new'
+    end
+  end
```

Comment:
Conditional looks great here supporting both a successful and non-successful registration.